koku/masu/test/database/test_aws_report_db_accessor.py

```diff
@@ -536,7 +536,7 @@ def test_populate_ocp_on_aws_cost_daily_summary_trino(self, mock_trino, mock_del
         with CostModelDBAccessor(self.schema, self.aws_provider.uuid) as cost_model_accessor:
             markup = cost_model_accessor.markup
             markup_value = float(markup.get("value", 0)) / 100
-            distribution = cost_model_accessor.distribution
+            distribution = cost_model_accessor.distribution_info.get("distribution_type", "cpu")
```

Comment:
Should this use a constant instead of "cpu"?

Reply:
Sure! I can throw the constant in the tests too.